scripts/openmc-track-to-vtk

```diff
@@ -74,26 +74,25 @@ def main():
             coords = h5py.File(fname).get('coordinates')
             n_points = coords.shape[0]
             for i in range(n_points):
-                points.InsertNextPoint(coords[i,:])
-                
+                points.InsertNextPoint(coords[i, :])
+
         # Create VTK line and assign points to line.
         line = vtk.vtkPolyLine()
         line.GetPointIds().SetNumberOfIds(n_points)
         for i in range(n_points):
             line.GetPointIds().SetId(i, point_offset+i)
-        
+
         cells.InsertNextCell(line)
         point_offset += n_points
     data = vtk.vtkPolyData()
     data.SetPoints(points)
     data.SetLines(cells)
-    
+
     writer = vtk.vtkXMLPPolyDataWriter()
-    writer.SetInput(data)
+    writer.SetInputData(data)
```

Comment:
Ah, ok. On 14.10, I guess they updated python-vtk to 6, hence the differences in APIs we're seeing. I used `vtk.vtkVersion.GetVTKMajorVersion()` to implement what you suggested, so now it should work for any version.